Playbooks/AS-Create-Opsgenie-Incident/azuredeploy.json

```diff
@@ -0,0 +1,310 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "AS-Create-Opsgenie-Incident", 
+        "description": "This playbook will create an incident in Opsgenie with the information from a Microsoft Sentinel incident.",
+        "prerequisites": "1. An Opsgenie account with either a Standard or Enterprise plan, along with an integration API key for this account. 2. A Microsoft Azure key vault containing the Opsgenie API key must also be set up. Support for the set up and configuration of each of these items can be found here: https://github.com/Accelerynt-Security/AS-Create-Opsgenie-Incident",
+        "lastUpdateTime": "2023-03-24T15:28:37Z",
+        "tags": ["Microsoft Sentinel", "Incident", "Opsgenie", "Integration"], 
+        "support": {
+            "tier": "developer"

```

Comment:
Support tier should be "Partner", assuming Accelerynt will support in case of any issue.